.vscode/settings.json

```diff
@@ -1,44 +0,0 @@
-{
```

Comment:
I don't understand why this file has been deleted

Reply:
Beats me.